prime-router/settings/prod/0003-wa-doh.yml

```diff
@@ -0,0 +1,21 @@
+---
+#
+# Organizations
+#
+- name: wa-doh
+  description: Washington State Department of Health
+  jurisdiction: STATE
+  stateCode: WA
+  receivers:
+    - name: elr
+      organizationName: wa-doh
+      topic: covid-19
+      jurisdictionalFilter: [ "matches(ordering_facility_state, WA)" ]
```

Comment:
Why not match on patient_state as well? - that's been our norm.
```suggestion
      jurisdictionalFilter: [ "orEquals(ordering_facility_state, WA, patient_state, WA)" ]
```

Reply:
Can do that. I was just following what was there.